design/resources.go

```diff
@@ -355,3 +355,25 @@ var _ = Resource("trackerquery", func() {
 	})
 
 })
+
+var _ = Resource("search", func() {
+	BasePath("/search")
+
+	Action("show", func() {
+		Routing(
+			GET("/:q"),
+		)
```

Comment:
thanks @DhritiShikhar for pointing out. done in https://github.com/almighty/almighty-core/pull/395/commits/6ed119d8efa1538288ee7ce806c6524998150444